Makefile

```diff
@@ -25,7 +25,7 @@ else
 	endif
 endif
 
-CLI_VERSION := $(if $(shell git describe --tags),$(shell git describe --tags),"UnknownVersion")
+CLI_VERSION ?= $(if $(shell git describe --tags),$(shell git describe --tags),"UnknownVersion")
```

Comment:
what does changing this line do?

Reply:
It makes it so that the CLI_VERSION is only defaulted to that value.  If you export `CLI_VERSION` or include it as an env var just for the make command it will take that value instead - I did it because I wanted an easy way to test the init download logic and give us a way to override the value if we wanted.